app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.ts

```diff
@@ -277,7 +329,7 @@ async function addEthereumChainHandler(
 
     // Once the network has been added, the requested is considered successful
     res.result = null;
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
You can find details about not using any [here](https://github.com/MetaMask/contributor-docs/blob/0a2b7c426d238adefadac7cc51eee20875d25785/docs/typescript.md#dont-use-any) we can switch to `unknown`

Reply:
`catch(error)` is accepting only `any` or `unknown`,; if I am using `unknown`,  it is not accepted in `end(error).` I typecasted it to `JsonRpcEngineCallbackError` in one `catch` section, but its not working for the other one as the `error.code` is being used in there.